### PR TITLE
Fix/refactorforsnsupdate

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
Adds example and uses APi key to install this integration.

### CODE REVIEW

File: aws_cloudtrail.trail.tf [hunk 1]
```diff
@@ -1,4 +1,4 @@
-resource aws_cloudtrail "trail" {
+resource "aws_cloudtrail" "trail" {
```
Comment (nimrodkor) [inline aws_cloudtrail.trail.tf:1]: These extra quote marks are not inline with the updated tf specifications/ conventions

**Review by nimrodkor — APPROVED** [review]
LGTM, small comment

**Review by mikeurbanski1 — APPROVED** [review]

File: aws_cloudtrail.trail.tf (hunk 1 above)
Comment (JamesWoolfenden) [reply aws_cloudtrail.trail.tf:1]: tf fmt

**Comment by JamesWoolfenden** [comment]
the pr hanging cos its circular so im merging it.

**Pull request merged**